pkgs/pre-commit-check/default.nix

```diff
@@ -89,7 +89,9 @@ pre-commit-hooks.lib.${system}.run {
               if [ "$PRE_COMMIT" = "1" ] \
               && [ -n "$PRE_COMMIT_FROM_REF" ] \
               && [ -n "$PRE_COMMIT_TO_REF" ]; then
-                ${config.hooks.commitizen.package}/bin/cz check --rev-range "$PRE_COMMIT_FROM_REF".."$PRE_COMMIT_TO_REF"
+                ${config.hooks.commitizen.package}/bin/cz check \
+                  --allowed-prefixes "Revert" \
+                  --rev-range "$PRE_COMMIT_FROM_REF".."$PRE_COMMIT_TO_REF"
```

Comment:
Will this cause commit statuses on features branches to be red until they are rebased?

I love preventing them from being merged, but not being able to quickly tell whether a PR is passing all other tests would be a pain.

Reply:
Beat you by 1 min hehe